### PR TITLE
Community docs

### LINKED ISSUE #42: Development workflow needs updating
This issue surrounds (1) dev workflow and (2) documentation.

For example, we didn't have a start command in package.json. 
I also just had to add a new section for development flow 

The documentation right now is pretty out of date. For example, we do have unit tests.

Can somebody familiar with Node CLI dev workflows help clean up the dev workflow and also the docs? Would be SUPER appreciated.

### PR DESCRIPTION
**Pull request by KmBarnett**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

This PR Adds contributing guide

## Added to documentation?

- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
+
 name: Bug report
 about: Create a report to help us improve
-title: ''
-labels: ''
+title: 'Bug: '
+labels: 'bug'~~~~~~~~
```
Comment (monarchwadia) [inline .github/ISSUE_TEMPLATE/bug_report.md]: What is `~~~~~` ?

**Review by monarchwadia — DISMISSED** [review]

**Comment by monarchwadia** [comment]
@KmBarnett one comment that may or may not need a fix.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (KmBarnett) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed

File: README.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ If it is your first time using `mintbean-cli`, first connect to GitHub with conf
 #### Suggested flow for new projects
 
 ```shell
-mint new <my-project>
-cd <my-project>
-mint repo --connect --push
+mint new
+cd
```
Comment (clairefro) [inline README.md]: add project name to clarify cd
```
mint new <my-project>
cd <my-project>
```

File: docs/how_to_contribute.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### Docs
+
+There is no such thing as too descriptive when it comes to documentation. If you feel like something in our docs could be better explained go for it.
+
+### Code
+
+##### 1. Project Templates
+
+Do you want a new template? Check out the adding new templates guide also in the Docs folder.
```
Comment (clairefro) [inline docs/how_to_contribute.md]: include a link to the guide

**Review by clairefro — CHANGES_REQUESTED** [review]
this PR was a lottttt more than updating the community docs lol

You mentioned you were experiencing trouble with `import/require` statements, and I see a mish-mash of both types in the code. 

If we decide to go the `import` route, maybe we try one of below:
```
TLDR;

Node >= v13

It's very simple in Node 13 and above. You need to either:

Save the file with .mjs extension, or
Add { "type": "module" } in the nearest package.json.
```

https://stackoverflow.com/questions/45854169/how-can-i-use-an-es6-import-in-node

File: README.md (hunk 2 above)
Comment (KmBarnett) [reply README.md]: done

File: docs/how_to_contribute.md (hunk 3 above)
Comment (KmBarnett) [reply docs/how_to_contribute.md]: done

**Review by clairefro — CHANGES_REQUESTED** [review]
Thanks for updating docs

We cannot approve this branch until it passes Buddy tests. Logs show that `jest` is not installed - that is why test is failing. Can you add jest and get tests to pass locally before pushing again?
 
image

**Review by clairefro — APPROVED** [review]

**Comment by clairefro** [comment]
Merging for now but it is urgent that we include tests in the Buddy pipeline soon

**Comment by clairefro** [comment]
Merging but it is urgent we write tests and include in Buddy pipeline soon
